### PR TITLE
Remove /myfeeds page, components, and deps

### PR DESCRIPTION
**Pull request by humphd**
We're moving to a Signup flow, with new UI/UX, and the current `/myfeeds` page can be retired.

This removes it, along with all the dependencies and child components I could find.  See if I've missed any.

### CODE REVIEW

File: src/web/package.json [hunk 1]
```diff
@@ -17,22 +17,17 @@
...
-    "swr": "^0.5.5",
-    "valid-url": "^1.0.9"
+    "swr": "^0.5.5"
   },
   "devDependencies": {
     "@types/node": "^14.14.37",
     "@types/react": "^17.0.3",
-    "@types/react-material-ui-form-validator": "^2.1.0",
     "@types/react-test-renderer": "^17.0.1",
-    "@types/valid-url": "^1.0.3",
```
Comment (PedroFonsecaDEV) [inline src/web/package.json:35]: All of these dependencies being removed were being used only on`/myfeeds` ?

Comment (humphd) [reply src/web/package.json:35]: Yeah, I believe so.  If we need them in new code, add them back, or flag now and I can leave them in.

Comment (PedroFonsecaDEV) [reply src/web/package.json:35]: Okay. let's remove them.

**Review by PedroFonsecaDEV — APPROVED** [review]

**Review by HyperTHD — DISMISSED** [review]

**Comment by humphd** [comment]
Rebased to pick up changes on master, can I get another review?

**Review by HyperTHD — APPROVED** [review]

**Review by manekenpix — APPROVED** [review]

**Pull request merged**